packages/editor/src/store/test/reducer.js

```diff
@@ -1580,9 +1580,28 @@ describe( 'state', () => {
 
 			expect( state ).toEqual( {
 				insertUsage: {},
+				isPublishSidebarEnabled: true,
 			} );
 		} );
 
+		it( 'should disable the publish sidebar', () => {
+			const original = deepFreeze( preferences( undefined, { } ) );
```

Comment:
>No filler spaces in empty constructs (e.g., {}, [], fn()).

https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#spacing

Need to find if possible to enforce by ESLint.

Reply:
https://github.com/eslint/eslint/issues/7927